frontend/packages/dev-console/integration-tests/features/Knative/channel-ODC-3225.feature

```diff
@@ -0,0 +1,148 @@
+Feature: Knative Eventing Channel Support
+    User should be able to experience the Knative Eventing Channel and associated features
+
+
+Background: 
+    Given user has installed the OpenShift Serverless Operator
+    And user has created knative serving
+    And user has create knative eventing
+
+
+@regression
+Scenario: Channel card on the +Add page
+    Given user is at Developer Perspective
+    When user goes to +Add page
+    Then user will see the Channel card on the +Add page
+
+
+@regression
+Scenario: Create InMemoryChannel
+    Given user is at Developer Perspective
+    When user goes to +Add page
+    And user clicks on the Channel card
+    And user selects InMemoryChannel from Type dropdown
```

Comment:
Default channel will be auto selected in the dropdown. Please include that information.